### PR TITLE
Implement populateTownSizeSelect

### PR DESCRIPTION
**Pull request by duncanspumpkin**

### CODE REVIEW

File: src/OpenLoco/Windows/TownList.cpp [hunk 1]
```diff
@@ -595,25 +595,25 @@ namespace OpenLoco::Ui::Windows::TownList
 
         static WindowEventList events;
 
+        constexpr string_id townSizeNames[9] = {
+            StringIds::tooltip_select_town_size,
```
Comment (AaronVanGeffen) [inline src/OpenLoco/Windows/TownList.cpp:599]: Why is this string id part of this array?

**Review by AaronVanGeffen — APPROVED** [review]
Works fine in its current state. Bit puzzled by the tooltip being included in the `townSizeNames` array.

File: src/OpenLoco/Windows/TownList.cpp (hunk 1 above)
Comment (duncanspumpkin) [reply src/OpenLoco/Windows/TownList.cpp:599]: I'm not entirely sure but think it's just the default value before selecting a size

Comment (AaronVanGeffen) [reply src/OpenLoco/Windows/TownList.cpp:599]: I think it defaults to three. Besides, it's a tooltip, right?

Comment (duncanspumpkin) [reply src/OpenLoco/Windows/TownList.cpp:599]: Hmm no idea then.

Comment (duncanspumpkin) [reply src/OpenLoco/Windows/TownList.cpp:599]: okay ive removed it.

**Review by AaronVanGeffen — APPROVED** [review]

**Pull request merged**